vendor/bat-native-ads/src/bat/ads/internal/client.cc

```diff
@@ -262,6 +262,17 @@ void Client::RemoveAllHistory() {
   SaveState();
 }
 
+std::string Client::GetVersionCode() const {
+  return client_state_->version_code;
+}
+
+void Client::SetVersionCode(const std::string& value) {
```

Comment:
Platform specific code belongs in the service layer, and should not be a part of the ads library. As now we are storing state for Android specific code